pkg/operator/ceph/cluster/cluster.go

```diff
@@ -460,6 +460,22 @@ func (c *cluster) preMonStartupActions(cephVersion cephver.CephVersion) error {
 // It gets executed right after the main mon Start() method
 // Basically, it is executed between the monitors and the manager sequence
 func (c *cluster) postMonStartupActions() error {
```

Comment:
Why the mon rotation  is required post upgrade?

What will happen if we will rotate first and then upgrade, or its internal ceph requirement?

Reply:
ceph requirement.